pkg/k8s/types/types.go

```diff
@@ -24,10 +24,11 @@ type CiliumEndpoint struct {
 	slim_metav1.TypeMeta
 	// +deepequal-gen=false
 	slim_metav1.ObjectMeta
-	Identity   *v2.EndpointIdentity
-	Networking *v2.EndpointNetworking
-	Encryption *v2.EncryptionSpec
-	NamedPorts models.NamedPorts
+	Identity       *v2.EndpointIdentity
+	Networking     *v2.EndpointNetworking
+	Encryption     *v2.EncryptionSpec
+	NamedPorts     models.NamedPorts
+	ServiceAccount string `json:"service-account,omitempty"`
```

Comment:
Nit. The `json` tag looks redundant here, but it doesn't hurt either.

Reply:
Each of the field structs seemed to have json tags as well. So added it to the outside one as well.

Reply:
Okay will remove then :)